include/rocksdb/env.h

```diff
@@ -936,7 +936,7 @@ class WritableFile {
   // size due to whole pages writes. The behavior is undefined if called
   // with other writes to follow.
   virtual Status Truncate(uint64_t /*size*/) { return Status::OK(); }
-  virtual Status Close() = 0;
+  virtual Status Close() { return Status::OK(); }
```

Comment:
Should this also return NotSupported?

Reply:
Oh, sorry I misunderstood that it was only for that file. Pushed again to resolve this.